src/Propel/Runtime/DataFetcher/PDODataFetcher.php

```diff
@@ -74,7 +74,7 @@ public function getStyle()
      *
      * @return array|null
      */
-    public function fetch($style = null)
+    public function fetch($style = null): ?array
```

Comment:
Many tests fails here:

```
Propel\Runtime\DataFetcher\PDODataFetcher::fetch(): Return value must be of type ?array, bool returned
```

I don't know what to do here.

Reply:
We should probably remove type for now and declare also bool value return